default/stringtables/en.txt

```diff
@@ -7809,10 +7801,23 @@ Direct Weapon
 PC_FIGHTER_HANGAR
 Hangar
 
+PC_FIGHTER_HANGAR_DESC
+'''Hangars are the storage, maintenance, and resupply areas for fighters.
+Since each hangar is built in support of other hangars on the ship, a ship is limited to one specific hangar type.
+
+After combat, fighters need to return to a hangar for repairs, fuel, and rearmament.
+Any fighter that is unable to dock at a hangar will likely be lost to the cold realities of space.
```

Comment:
Rephrase without flavour text... "If a carrier is destroyed in combat, any of its fighters that survive the battle will be unable to dock after combat, and will also be lost."